Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/MSBuildForUnity.Common.props.template

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <PropertyGroup>
+    <MSBuildForUnityVersion>0.8.0</MSBuildForUnityVersion>
+    <MSBuildForUnityGeneratedOutputDirectory><!--GENERATED_OUTPUT_DIRECTORY_TOKEN--></MSBuildForUnityGeneratedOutputDirectory>
```

Comment:
Can this somehow be generated? It seems like it would be very easy for this to get out of sync with the package version (assuming it's supposed to be in sync)

Reply:
I thought about it, I am not sure. We have 3 places where we gotta keep versions in sync:
- This common props file
- package.json for the UPM package
- The NuGet build project

Any ideas? I also think we can fix this in a subsequent release

Reply:
Filed #56